source/declarations.tex

```diff
@@ -3017,25 +3017,29 @@
 
 \pnum
 \indextext{overloading!using-declaration and}%
+\begin{note}
 For the purpose of forming a set of candidates during overload resolution,
 the functions that are
 introduced by a \grammarterm{using-declaration} into a derived class
-are treated as though they were members of the derived class. In
-particular, the implicit \tcode{this} parameter shall be treated as if
-it were a pointer to the derived class rather than to the base class.
+are treated as though they were members of the derived class\iref{class.member.lookup}. In
+particular, the implicit object parameter is treated as if
+it were a reference to the derived class rather than to the base class\iref{over.match.funcs}.
 This has no effect on the type of the function, and in all other
 respects the function remains a member of the base class.
-Likewise, constructors that are introduced by a \grammarterm{using-declaration}
+A member of a derived class is sometimes preferred to a member of a base class
+if they would otherwise be ambiguous\iref{over.match.best}.
```

Comment:
This actually only applies to the constructor case, right? (That is, what's now the next paragraph.)

Reply:
I'm not sure. With using-declarations and multiple inheritance etc., maybe member name lookup can yield a lookup set that contains both members of a derived class and of a base class (if name lookup could find the members of the base class via some other path). In that case, that note would also apply in that overload resolution gets to choose a unique result.

Reply:
The above "In particular, the implicit object parameter is treated as if it were a reference to the derived class" suggests that we don't take derived-to-base conversions on the implied object parameter into account. And actually, I'm not seeing anywhere in [over.match.funcs] that makes this clear; we are just told "X is the class of which the function is a member". So I think that part here is not duplicating other normative wording either.

So I'm not sure there's really all that much normative duplication here.

Reply:
@zygoloid, we have in p4 in [over.match.funcs] "For non-conversion functions introduced by a using-declaration into a derived class, the function is considered to be a member of the derived class for the purpose of defining the type of the implicit object parameter."  What's missing in that sentence that would be added by a normative "In particular ..." sentence?